HACKING.txt

```diff
@@ -25,6 +25,21 @@ tox on PyPI: https://pypi.org/project/tox/
 
   Alternatively, create a writeable fork on GitHub and clone it.
 
+- Make sure that `tox` is installed, either in your path, or locally. Examples
+  below assume that `tox` was installed with:
+
+    $ pip3 install --user tox
+    $ export TOX=$(python3 -c 'import site; print(site.USER_BASE + "/bin")')/tox
+
+  Feel free to use whatever your favourite method for installation is, one
+  popular one is `pipx`: https://github.com/pipxproject/pipx which then may be
+  easily added to your shell's PATH variable so you may execute tox without
+  providing a full path
```

Comment:
Do we really want to get into recommending tools for installing packages? _/me backs away slowly._

I prefer to link to the [PyPA's Guides](https://packaging.python.org/guides/) and call it soup.

Reply:
You were the one that argued for having instructions for users.

Reply:
Sure, but not going into details of using pipx or how to invoke tox according to where it is installed (i.e., global, user, virtual environment).